dockers/actions-plan-preview/main.go

```diff
@@ -109,8 +112,31 @@ func main() {
 		}
 	}
 
+	// Find comments we sent before
+	comment, err := findLatestPlanPreviewComment(ctx, ghGraphQLClient, event.Owner, event.Repo, event.PRNumber)
+	if err != nil && !errors.Is(err, errNotFound) {
+		log.Fatal(err)
+	}
```

Comment:
```suggestion
	if err != nil && !errors.Is(err, errNotFound) {
		log.Printf("Failed to find the previous comment to minimize (%v)", err)
	}
```

Reply:
If we make the changes you pointed out below, I don't think we will see the log in case of errNotFound. How about turning off the `&& !errors.Is(err, errNotFound)`?

Reply:
I get your point. Ok so let's remove that check.

Reply:
I sent another suggestion based on your idea. So let me close this one.
https://github.com/pipe-cd/pipecd/pull/3126/files#r793278458